### PR TITLE
new YAMLs TJB

### PR DESCRIPTION
**Pull request by tjbarnum13**
new YAMLs for quinoline, pyridazine, isoquinoline, and imidazole

### CODE REVIEW

File: data/isoquinoline.yml [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+chi_bb: 0.1304
+chi_cc: 3.3810
+chi_ab: 2.81
+u_a: 2.3602
+u_b: 0.9051
+rep: Ir
+doi: [10.1016/S0022-2852(02)00020-6]
+name: isoquinoline
+smiles: C1=CC=C2C=NC=CC2=C1
+notes: C9H7N
```
Comment (bchangala) [inline data/isoquinoline.yml]: change to formula

File: data/pyridazine.yml [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+phi_JK: -0.000252E-6
+phi_K: 0.002081E-6
+chi_aa_1: -4.6411
+chi_bb_1: 1.3711
+chi_cc_1: 3.2700
+chi_ab_1: 2.709
+chi_aa_2: -4.6411
+chi_bb_2: 1.3711
+chi_cc_2: 3.2700
+chi_ab_2: 2.709
```
Comment (bchangala) [inline data/pyridazine.yml]: I think the off-diagonal chi_ab constants for 1 and 2 should have signs opposite of each other, but which one is + and which one is - is not determined.

**Review by bchangala — CHANGES_REQUESTED** [review]
See note on sign of pyridazine chi's

File: data/pyridazine.yml (hunk 2 above)
Comment (tjbarnum13) [reply data/pyridazine.yml]: great - that sounds reasonable to me. i knew you would have the answer bryan!

**Review by bchangala — APPROVED** [review]

**Pull request merged**